test/NPacMan.Game.Tests/AsciiTestSettingBuilder.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace NPacMan.Game.Tests
+{
+    class AsciiTestSettingBuilder
+    {
+        private readonly TestGameSettings _gameSettings;
+        private CellLocation _topLeft;
+
+        public AsciiTestSettingBuilder(TestGameSettings gameSettings)
+        {
+            _gameSettings = gameSettings;
+        }
+        internal void CreateBoard(params string[] rows)
+        {
+            rows = rows.Select(RemovePadding).ToArray();
+
+            _topLeft = CalculateTopLeftFromPacMan(rows);
+
+            RemoveCurrentSettings();
+
+            var currentCell = _topLeft;
+            foreach (var row in rows)
+            {
+                CreateRow(row, currentCell);
+                currentCell = currentCell.Below;
+            }
+        }
+
+        internal void AssertBoard(Game game, params string[] rows)
+        {
+            rows = rows.Select(RemovePadding).ToArray();
+
+            var currentCell = _topLeft;
+            foreach (var row in rows)
+            {
+                AssertRow(game, row, currentCell);
+                currentCell = currentCell.Below;
+            }
+        }
+
+        private void RemoveCurrentSettings()
+        {
+            _gameSettings.GhostHouse.Clear();
```

Comment:
Should we also remove coins etc

Reply:
Would be a bit iffy as we've got go put a coin FarAway from where we're playing so we don't complete the level.